units/systemd-timesyncd.service.in

```diff
@@ -12,7 +12,8 @@ ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
 RequiresMountsFor=/var/lib/systemd/clock
-After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service
+Requires=dev-rtc.device
+After=systemd-remount-fs.service systemd-tmpfiles-setup.service systemd-sysusers.service dev-rtc.device
```

Comment:
I don't like this I must say. There are devices lacking an RTC around, and this means on those systems we'll wait for it forever. If you are concerned about racing against /dev/rtc showing up here at boot, I'd suggest we simply retry in regular intervals to sync to RTC until we manage to find the device. I think this would make things more robust in the long run.